_platforms/mlops.md

```diff
@@ -4,4 +4,16 @@ platform-name: MLOps
 platform-image: /assets/img/platforms/mlops_logo.png
```

Comment:
Can we use Laurence's updated MLOps logo. Can get from the MLOps repo.

Reply:
Could we remove the black borders from the images on these pages? I think visually it'd be much cleaner without them